modules/installation-aws-iam-user.adoc

```diff
@@ -21,7 +21,18 @@ procedure in the AWS documentation, set the following options:
 . Specify the IAM user name and select `Programmatic access`.
 
 . Attach the `AdministratorAccess` policy to ensure that the account has
-sufficient permission to create the cluster.
+sufficient permission to create the cluster. Using this policy is a best practice
+because it provides the cluster with the ability to grant additional credentials,
+and that permission allows it to grant only the required permissions to each
+{product-title} component.
```

Comment:
Suggest: "because it provides the cluster with the ability to grant credentials to each {product-title} component, with fine grained permissions allowing only what that component needs."

Reply:
Thank you @dgoodwin! I've updated the section.